main/sources/fullImport.integration.ts

```diff
@@ -291,3 +298,591 @@ const expected: BKProject = {
     },
   ],
 };
+
+const expectedSAB = {
+  name: testPaths.exampleSABProjectName,
+  folderPath: testPaths.exampleSABProjectPath,
+  sourceType: 'scriptureAppBuilder',
+  books: [
+    {
+      name: 'Mat',
+      chapters: [
+        {
+          name: '1',
+          audio: {
+            files: [
+              {
+                filename: 'C:\\Audacity\\BK Test Matthew 1.mp3',
```

Comment:
This might work when you run it on your local machine if you happen to have the files downloaded but it won't work when run on GHA (GitHub Actions) on our build step. Instead add the files to the repo if needed and reference them there.

Reply:
Thanks for the rebase. I've fixed this now so the test passes.